functests/08-test-conditions-node-labeller.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+[ -z ${KV_NAMESPACE} ] && exit 127
+
+RET=0
+TEST_NS="${KV_NAMESPACE}"
+
+exit 0
+
+oc create -n ${TEST_NS} -f "${SCRIPTPATH}/node-labeller-unversioned-cr.yaml" || exit 2
+# TODO: SSP-operator needs to improve its feedback mechanism
+# fetching node-labeller images may take a while
+
+wait_node_labeller_running ${TEST_NS} 5 60
+#wait for ssp operator to set proper conditions
+sleep 15s
+
+if [ $(oc get KubevirtNodeLabellerBundle -o json | jq  '.items[0].status.conditions[] | select((.type=="Progressing") and (.status="True")) | .type' | wc -l) -eq 0 ]; then
+		RET=0
```

Comment:
the logic is confusing to me. You should either:
- assume the test will run OK, check conditions, bail out with failure at first unexpected state and return error
- assume the test will fail, check conditions, only iff all of them are the expected state then return success
You seem to do a strange mix of the two in the code